readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md

```diff
@@ -0,0 +1,208 @@
+﻿# Opensds Telemetry with LVM: User Guide
+
+
+## Getting Started!
+This guide is a "Hello World"-style tutorial which shows how to install, configure, and use Telemetry feature in a simple opensds with lvm  setup.
+By following this guide you will be able to install opensds with telemetry tools, create test volumes, create sample grafana dashboard, create sample alert rules and view dashboards.
+
+## Deploying Opensds with Telemetry
+Follow this opensds wiki link to install opensds with telemetry and 
+***Pay special attention to the section below before starting the deploy operation!***
+
+**Download opensds-installer code**
+
+    
+    git clone https://github.com/opensds/opensds-installer.git
+    cd opensds-installer
+    # Checkout the latest release, give master by defualt or specfy the branch 
+    git checkout master
+   
+    
+  **How to enable Telemetry installation**
+  
+   Update the file : ansible/group_vars/telemetry.yml to install telemetry tools
+   
+    
+    # Do you need to install telemetry tools?
+    install_telemetry_tools: true
+    
+Detailed instruction is in this link
+
+- [OpenSDS Local Cluster Installation through Ansible On Ubuntu](https://github.com/opensds/opensds/wiki/OpenSDS-Cluster-Installation-through-Ansible) (Recommended)
+
+### Make environment ready for metric collection
+
+**export required OPENSDS environment variables**
+```
+export OPENSDS_ENDPOINT=http://{{ apiserver_cluster_ip }}:50040
```

Comment:
Here is not apiserver_cluster_ip, instead it's host_ip

Reply:
Changed, Thanks